src/SprykerSdk/Zed/AppSdk/Business/AsyncApi/Builder/AsyncApiBuilder.php

```diff
@@ -88,6 +98,26 @@ public function addAsyncApiMessage(AsyncApiRequestTransfer $asyncApiRequestTrans
         return $asyncApiResponseTransfer;
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\AsyncApiRequestTransfer $asyncApiRequestTransfer
+     *
+     * @return true|false
+     */
+    protected function isPropertyOptionEmpty(AsyncApiRequestTransfer $asyncApiRequestTransfer): bool
+    {
+        return $asyncApiRequestTransfer->getPayloadTransferObjectName() === null && count($asyncApiRequestTransfer->getProperties()) === 0;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\AsyncApiRequestTransfer $asyncApiRequestTransfer
+     *
+     * @return true|false
```

Comment:
```suggestion
     * @return bool
```
This is almost all the time the same as you have in your PHP return type `): bool`.

Reply:
I used bool return type before, but when I check with composer cs-check or composer-stan then it gave some error so after some research I used true|false.